metrics/stmt.current/current.go

```diff
@@ -0,0 +1,168 @@
+package stmt
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sort"
+	"strconv"
+
+	"github.com/cashapp/blip"
+)
+
+const (
+	DOMAIN = "stmt.current"
+	query  = `SELECT TIMER_WAIT FROM performance_schema.events_statements_current 
+		WHERE END_EVENT_ID IS NULL 
+		AND EVENT_NAME NOT LIKE ('statement/com/Binlog%')`
+	OPT_THRESHOLD = "threshold"
```

Comment:
Let's call this option `slow-threshold` to give a little more context (because just "threshold" in the stmt.current domain could be a threshold for anything).